src/PowerShellEditorServices/Services/Symbols/SymbolsService.cs

```diff
@@ -284,7 +263,12 @@ static string[] GetIdentifiers(string symbolName, SymbolType symbolType, Command
                 return null;
             }
 
-            return AstOperations.FindReferencesOfSymbol(file.ScriptAst, foundSymbol).ToArray();
+            if (file.References.TryGetReferences(foundSymbol.SymbolName, out ConcurrentBag<SymbolReference> references))
```

Comment:
Filter on symbol type. Will currently match ex. a similar function name and enum/class name